tsrc/workspace/cloner.py

```diff
@@ -9,9 +11,12 @@
 
 
 class Cloner(tsrc.executor.Task[tsrc.Repo]):
-    def __init__(self, workspace_path: Path, *, shallow: bool = False) -> None:
+    def __init__(
+        self, workspace_path: Path, *, shallow: bool = False, remote_name: Union[str, None] = None
```

Comment:
Use `Optional[str]` instead

Reply:
Also fixed this in two other places where I should've used `Optional[str]`.